src/Silverback.Core/Messaging/Configuration/SilverbackBuilderWithLogLevelsExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) 2020 Sergio Aquilini
+// This code is licensed under MIT license (see LICENSE file for details)
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Silverback.Messaging.Configuration
+{
+    /// <summary>
+    /// Extension to configure the loglevels of Silverback.
+    /// </summary>
+    public static class SilverbackBuilderWithLogLevelsExtensions
+    {
+        /// <summary>
+        /// Configure the loglevels that should be used to log events.
+        /// </summary>
+        /// <param name="silverbackBuilder">An <see cref="ISilverbackBuilder"/>.</param>
+        /// <param name="configure">The action that configures the loglevels.</param>
+        /// <returns>The <see cref="ISilverbackBuilder"/>.</returns>
+        public static ISilverbackBuilder WithLogLevels(this ISilverbackBuilder silverbackBuilder, Action<ILogLevelConfigurator> configure)
+        {
+            if (configure is null)
```

Comment:
Please use `Check.NotNull` as in the rest of the solution.

Reply:
I fixed this.